ssa.go

```diff
@@ -1163,7 +1163,11 @@ func (e *ssaEvent) string(format []string) string {
 			var i *int
 			switch attr {
 			case ssaEventFormatNameLayer:
-				i = e.layer
+				if e.layer == nil {
```

Comment:
Could you move this `if` line 1178 so that it "protects" all *int ?

Also, could you make that for `ssaEventFormatNameMarked` there's `Marked=0` if value is nil?

Reply:
sure, i've change it on the last  commit.